gsonpath/src/main/java/gsonpath/GsonPath.java

```diff
@@ -10,6 +10,10 @@
 public class GsonPath {
     private static final String FACTORY_IMPLEMENTATION_SUFFIX = "Impl";
 
+    public GsonPath() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)